modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/DefaultComponent.java

```diff
@@ -63,6 +65,10 @@ public void deactivate(ComponentContext context) {
         setModifiedNow();
     }
 
+    protected void addRuntimeMessage(Level level, String message) {
+        Framework.getRuntime().getMessageHandler().addMessage(level, message, ComponentManagerImpl.class.getName());
```

Comment:
Shouldn't be getClass instead of ComponentManagerImpl.class?

Reply:
These are still registration errors, so if we want to cleanup them up on hot reload, they should be linked to the ComponentManager marker

Reply:
source removed --> resolve